src/main/logutils.ts

```diff
@@ -330,35 +381,19 @@ function handleArenaStartLine (line: LogLine): void {
 function handleArenaStopLine (line: LogLine): void {
     if (!recorder.isRecording) return; 
 
-    if (playerCombatant) {
-        metadata.playerName = playerCombatant.name;
-        metadata.playerRealm = playerCombatant.realm;
-        metadata.playerSpecID = playerCombatant.specID;        
-    }
-
-    let duration; 
+    let duration;
+    videoStopDate = line.date();
     
     // Helpfully ARENA_MATCH_END events contain the game duration. Solo shuffle
```

Comment:
Is there any value in keeping this? We can just use the date events from logs and then we don't need any special handling for solo shuffle?

Reply:
For PvP there might not be, no - I think only Mythic+ _requires_ the duration from the logs to be saved because it's always different from the video duration due to overrun, but I don't think any raid or PvP activity has this issue. I'll remove it!